metrics-recorder-text/src/lib.rs

```diff
@@ -191,7 +194,13 @@ impl MetricsTree {
 
 impl Into<String> for TextRecorder {
```

Comment:
Why not implement `From<TextRecorder>` for String? This also provides `Into<String> for TextRecorder`.

Reply:
Ah yeah, good point.  Will change.